### ISSUE TITLE
New user cannot login multi-cluster environment

### ISSUE THREAD

**Issue by wenxin-01** [kubesphere/kubesphere#4296]
**Versions Used**
KubeSphere:  `ks3.2-alpha.0`

![image](https://user-images.githubusercontent.com/68640256/135036467-45c19bf8-cf07-4f32-be04-5a4eb49b1837.png)

**How To Reproduce**
1. create new user 'wx2' in multi-cluster environment
2. user user wx2 login ks
3. See error
![image](https://user-images.githubusercontent.com/68640256/135035537-cb15f5f4-3582-47f2-b360-c111494689f1.png)
![image](https://user-images.githubusercontent.com/68640256/135035695-8613065c-a7c8-4b03-9d5b-ab9826566206.png)

**Expected behavior**
login successed

/kind bug
/priority High
/milestone 3.2
/assign @wansir

**Comment by iawia002** [comment]
> @iawia002 This issue already fixed in #4254
> 
> The object is registered to [runtime.Scheme](https://github.com/kubesphere/kubesphere/blob/master/staging/src/kubesphere.io/api/types/v1beta1/register.go#L78), which used in controller-runimt.Client. It can be verified in recent nightly build.

Nope, #4254 doesn't fix it, I know the scheme is registered, I am also very curious about this issue, it is not 100% reproducible.
